node/subsystem-util/src/lib.rs

```diff
@@ -292,15 +300,15 @@ pub fn signing_key(validators: &[ValidatorId], keystore: &KeyStorePtr) -> Option
 /// relay chain block.
 pub struct Validator {
 	signing_context: SigningContext,
-	key: ValidatorPair,
+	key: ValidatorId,
```

Comment:
this is the crux of the changes -- previously we stored both public and private keys here, now only public and fetch (sign with) the private key from the keystore

the rest are tests fixes